tests/files/afval.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "afvalwegingen",
+  "type": "dataset",
+  "title": "Afvalwegingen",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "containers",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["id", "schema"],
+        "display": "id",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "description": "Container-ID"
+          },
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/schema"
+          },
+          "cluster": {
+            "type": "string",
+            "relation": "afvalwegingen:clusters",
```

Comment:
Vraag, voor mijn beeld, ik was in de veronderstelling dat je het onderstaande als n:m definitie moet hanteren. Of klopt het zoals het in afwegingen staat en is het onderstaande ergens anders gedefinieerd?

`"refereertaanreferentiepunten": {
    "type": "array",
    "items": {
        "type": "string",
        "relation": "meetbouten:referentiepunten"
    },
    "description": "De referentiepunten waar de metingen aan worden opgehangen"

}`

Reply:
Die relatie in afvalwegingen is een 1-N relatie. Dus een container staat altijd in 1 cluster (en een cluster heeft meerdere containers). Voor de referentiepunten kan een meting refereren aan meerdere referentiepunten en kan een referentiepunt over meerdere metingen gaan, dus N-M.